doc/build.info

```diff
@@ -0,0 +1,64 @@
+{-
+ use File::Spec::Functions qw(:DEFAULT abs2rel rel2abs);
+ use File::Basename;
+
+ foreach my $section ((1, 3, 5, 7)) {
+     my @htmlfiles = ();
+     my @manfiles = ();
+     my %podfiles =
+         map { $_ => 1 } glob catfile($sourcedir, "man$section", "*.pod");
+     my %podinfiles = 
+         map { $_ => 1 } glob catfile($sourcedir, "man$section", "*.pod.in");
+
+     foreach (keys %podinfiles) {
+         (my $p = $_) =~ s|\.in$||i;
+         $podfiles{$p} = 1;
+     }
+
+     foreach my $p (sort keys %podfiles) {
+         my $podfile = abs2rel($p, $sourcedir);
+	 my $podname = basename($podfile, '.pod');
+	 my $podinfile = $podinfiles{"$p.in"} ? "$podfile.in" : undef;
```

Comment:
Indentation is strange here

Reply:
Tabs...  and github now doing the M$ thing (4 column tabs)

Reply:
This still seems to be a problem

Reply:
*Now* it's done.